pom.xml

```
@@ -738,7 +738,7 @@ under the License.
       org.apache.log;version="[1.0.1, 1.0.1]";resolution:=optional,
       org.apache.log4j;version="[1.2.15, 2.0.0)";resolution:=optional,
       org.apache.logging.log4j;version="[2.0, 4.0)";resolution:=optional,
-      org.slf4j;version="1.7.0, 3.0";resolution:=optional
+      org.slf4j;version="[1.7.0, $(versionmask;+;${slf4j.version}))";resolution:=optional
```

```suggestion
      org.slf4j;version="[1.7, 3)";resolution:=optional
```

I would prefer to abstain from macros. When SLF4J 3.x comes out, it could be compatible with 2.x or not. Replacing `3` with `4` in this version range will need a human to approve it.